tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -62,3 +65,7 @@ install-tarball:
 
 image: kata-tarball
 	$(MK_DIR)kata-deploy-build-and-upload-image.sh $(CURDIR)/kata-static.tar.xz
+
+sev-tarballs: kernel-tarball qemu-tarball rootfs-initrd-tarball shim-v2-tarball ovmf-tarball
+
+sev: sev-tarballs-parallel merge-builds
```

Comment:
This looks weird to me.
Do we already scripts for building sev?  Do we need a specific target for building sev?

Reply:
I added a new target for SEV since I didn't think other use cases would need OVMF. I am open to adding ovmf to other images, but just was trying to avoid potentially unnecessary dependencies. I definitely see how this fits into the discussion on #3992, so I am curious how that pans-out.